napari/_qt/exceptions.py

```diff
@@ -37,9 +42,13 @@ def handle(
         tb : TracebackType
             The traceback object associated with the error.
         """
-        # etype.__module__ contains the module raising the error
-        # Custom exception classes can have different behavior
-        # can add custom exception handlers here ...
-        text = "".join(traceback.format_exception(etype, value, tb))
-        logging.error("Unhandled exception:\n%s", text)
         self.error.emit((etype, value, tb))
+        if session_is_interactive() or os.getenv("NAPARI_SHOW_CONSOLE_ERRORS"):
```

Comment:
Thanks for adding this. I'm sorry for missing something really basic here, but I'm having a hard time turning this off!!

If I run
```
export NAPARI_SHOW_CONSOLE_ERRORS=0
```
in the command line, and then run
```python
        print(os.getenv("NAPARI_SHOW_CONSOLE_ERRORS"), bool(os.getenv("NAPARI_SHOW_CONSOLE_ERRORS")))
# 0, True
```
Just before this call, then I get a value of `0`, but calling bool gives `True`. Is that maybe because it is treating that `0` like a string instead of an int?

Similarly if I do
```
export NAPARI_SHOW_CONSOLE_ERRORS=False
```
I then get
```python
print(os.getenv("NAPARI_SHOW_CONSOLE_ERRORS"), bool(os.getenv("NAPARI_SHOW_CONSOLE_ERRORS")))
# False, True
```
where my `False` is getting interpreted as True too!

Looks like in #1510 @pwinston did an explicit check `=="0"` and `=="1"` and I think that worked for me https://github.com/napari/napari/pull/1510/files#diff-33a998e174214127b7c29099bc60f479R126-R131. We might want to consider that approach here too.

Reply:
hah.  sorry... sure I can do that.   But you can also use `unset NAPARI_SHOW_CONSOLE_ERRORS`

Reply:
> But you can also use unset NAPARI_SHOW_CONSOLE_ERRORS

Ah nice, thanks for the tip!! I didn't know this. I think having both `=='0'` and the unset approach will be good, thanks!!

Reply:
I'll add that the behaviour is now perfect and the toggling is very nice!!